src/rgw/rgw_file.h

```diff
@@ -2287,7 +2287,7 @@ class RGWStatBucketRequest : public RGWLibRequest,
   }
 
   void send_response() override {
-    bucket->get_creation_time() = get_state()->bucket_info.creation_time;
+    bucket->get_creation_time() = get_state()->bucket->get_info().creation_time;
```

Comment:
It could be.   The original problem was that it's a sub-field in RGWBucketEnt, which partially over-laps with RGWBucketInfo, so we can't actually expose either of those.  I'll add this to the list to look at in the next patch, since it's not actually added in this one.